doc/runtime.md

```diff
@@ -64,6 +67,10 @@ Here is an example that uses the `runtime` overlay to launch all of the required
           # you probably want this one too, although it's not demonstrated
           # in this example
           cardano-transaction-lib.overlays.purescript
+          # This one is optional. If you set `ctl-server.enable = true;`
+          # in the runtime config as shown below, you must enable this
+          # overlay. `ctl-server` itself is **only** required when using
+          # CTL's `applyArgs` effect
```

Comment:
Sorry if I'm misunderstanding, but this is supposed to be followed by `cardano-transaction-lib.overlays.ctl-server` I think?

Reply:
Good catch, you're right